cgi/mapcache.c

```diff
@@ -158,6 +162,16 @@ static void fcgi_write_response(mapcache_context_fcgi *ctx, mapcache_http_respon
   }
 }
 
+// Replace all occurrences of substr in string
+char* str_replace_all(apr_pool_t* pool, const char* string,
+    const char* substr, const char* replacement)
+{
+    char* replaced = apr_pstrdup(pool, string);
+    while (strstr(replaced, substr)) {
+        replaced = mapcache_util_str_replace(pool, string, substr, replacement);
+    }
+    return replaced;
+}
 
```

Comment:
No need for this, see other comment.

Reply:
I'm not sure how I missed `mapcache_util_str_replace_all` - I was hoping that function existed already somewhere!